plugins/covid/models.py

```diff
@@ -0,0 +1,15 @@
+"""
+Models for the Covid plugin
+"""
+from django.db import models
+
+
+class CovidDashboard(models.Model):
+    """
+    Stores figures for the Covid 19 Dashboard in the
+    database when they take a significant amount of time
+    to calculate.
+    """
+    patients_tested = models.IntegerField()
```

Comment:
I think we should add auto add fields

Reply:
Not sure there's any point. Created is always arbitrarily 12 minutes past the last hour b/c the cron job and the delete all. We only need one row in this table, we're not storing history.